frontend/src/metabase/nav/components/WhatsNewNotification/WhatsNewNotification.unit.spec.tsx

```diff
@@ -47,16 +49,25 @@ const setup = ({
   const versionMock = createMockVersion({ tag: currentVersion });
 
   const [latest, ...older] = versions;
-  const mockSettings = createMockSettingsState({
+  const mockedSettings = mockSettings({
     version: versionMock,
     "version-info": createMockVersionInfo({ latest, older }),
     "last-acknowledged-version": lastAcknowledged,
+    "application-name": isWhiteLabeling ? "My App" : "Metabase",
+    "token-features": createMockTokenFeatures({
+      whitelabel: true,
```

Comment:
since you also conditionally set the `application-name`, you should do the same here.

```suggestion
      whitelabel: isWhiteLabeling,
```